python/packages/ai/teams/ai/action/default_action_types.py

```diff
@@ -0,0 +1,16 @@
+"""
+Copyright (c) Microsoft Corporation. All rights reserved.
+Licensed under the MIT License.
+"""
+
+from enum import Enum
+
+
+class DefaultActionTypes(str, Enum):
+    UNKNOWN_ACTION = "__UnknownAction__"
```

Comment:
JS SDK has **3** underscores in the name, should we align the name?

Reply:
the C# implementation supports `void` https://github.com/microsoft/teams-ai/blob/main/dotnet/packages/Microsoft.TeamsAI/Microsoft.TeamsAI/AI/Action/ActionHandler.cs#L28 and I think it makes sense to support and default to `false`